BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m

```diff
@@ -224,6 +221,14 @@ - (void)performRequestWithCached:(BOXFileBlock)cacheBlock
     [self performRequestWithCompletion:refreshBlock];
 }
 
+- (void)setRepresentationRequestOptions:(NSArray *)representationOptions {
+    [self.repsHelpers setRepresentationRequestOptions:representationOptions];
+}
+
+- (NSString *)formatRepresentationRequestHeader {
+    return [self.repsHelpers formatRepresentationRequestHeader];
+}
+
```

Comment:
Do we need this functions or can we invoke the helper directly.  If we need them, perhaps they ought to be in some protocol.